Core/include/Acts/Utilities/AlgebraHelpers.hpp

```diff
@@ -248,4 +249,41 @@ constexpr T safeExp(T val) noexcept {
   return std::exp(val);
 }
 
+/// @brief Map the indices of the lower triangular part of a symmetric N x N matrix
+///        to an unrolled vector index.
+/// @param i The row index of the symmetric matrix
+/// @param k The column index of the symmetric matrix
+template <unsigned N>
+constexpr unsigned vecIdxFromSymMat(unsigned i, unsigned k)
```

Comment:
Doesn't this depend on column/row major?

also: I think `unsigned` is rather an unusual type in ACTS